### PR TITLE
Add Github integration

### PR DESCRIPTION
**Pull request by mejo-**
TODO:
* [x] Behat: figure out why some (but not all) tests that do OCS API calls to circles fail
* [x] Cypress: make tests run. Apparently the whole Collectives app is missing currently
* [x] Cypress: enable recording to the Cypress cloud (?)
* [x] Change links to Gitlab to Github (e.g. in `appinfo/info.xml`)

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+onlyLabels: 
+  - "needs info"
```
Comment (marcelklehr) [inline .github/stale.yml:7]: Nice! I wish we'd use that on server

Comment (mejo-) [reply .github/stale.yml:7]: Indeed. I copied it over from Text :relaxed:

File: cypress.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "https://localhost:8081/index.php/",
+  "baseUrl": "http://localhost:8081/index.php/",
   "viewportWidth": 1280,
   "viewportHeight": 720,
   "experimentalSessionAndOrigin": true
```
Comment (juliushaertl) [inline cypress.json:5]:
```suggestion
  "experimentalSessionAndOrigin": true,
  "projectId": "b6myzg"
```

Created a project and added the secret to the repo.

Comment (juliushaertl) [reply cypress.json:5]: https://cloud.cypress.io/projects/b6myzg/runs

Comment (mejo-) [reply cypress.json]: Thanks @juliushaertl - added.

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: bug
+---
```
[the remark this answers is not in the document]
Comment (mejo-) [reply .github/ISSUE_TEMPLATE/Bug_report.md:5]: Great idea, but I would like to postpone this to a later point. Would prefer to get this PR merged first :grimacing:

**Review by juliushaertl — APPROVED** [review]
Looks good, cypress failure only seems on stable25 so maybe that is flaky somehow?

**Comment by mejo-** [comment]
> Looks good, cypress failure only seems on stable25 so maybe that is flaky somehow?

Yeah, one is flaky and I hope to have it fixed now. The other one I'm looking into right now :blush:

**Comment by juliushaertl** [comment]
Btw. should we enable the cypress comments on the pull requests here as well so that it is directly visible which cypress tests fail?

**Pull request merged**